test/stdcpp/src/vector.cpp

```diff
@@ -1,17 +1,60 @@
 #include <vector>
 
+extern int vectorTest_numAllocated;
+
+struct NotPOD
+{
+    NotPOD() = delete;
+    NotPOD(int a)
+    {
+        for (int i = 0; i < 5; ++i)
+            x[i] = a + i + 1;
+        wasInit = true;
+        ++vectorTest_numAllocated;
+    }
+    NotPOD(const NotPOD& rh)
+    {
+        for (int i = 0; i < 5; ++i)
+            x[i] = rh.x[i];
+        wasInit = rh.wasInit;
+        if (rh.wasInit)
+            ++vectorTest_numAllocated;
+    }
+    NotPOD(NotPOD&& rh) noexcept
+    {
+        for (int i = 0; i < 5; ++i)
+            x[i] = rh.x[i];
+        wasInit = rh.wasInit;
+        rh.wasInit = false;
+    }
+    ~NotPOD()
+    {
+        if (wasInit)
+            --vectorTest_numAllocated;
+    }
+
+    int x[5] = { 1, 2, 3, 4, 5 };
```

Comment:
`test\stdcpp\src\vector.cpp(36) : error C2536: 'NotPOD::NotPOD::x' : cannot specify explicit initializer for arrays`

Reply:
yep, bloody ancient compilers!